scale/job/execution/tasks/exe_task.py

```diff
@@ -86,15 +86,14 @@ def determine_error(self, task_update):
 
         raise NotImplementedError()
 
-    @abstractmethod
     def populate_job_exe_model(self, job_exe):
```

Comment:
The new pull task doesn't do anything to populate the model, so instead of forcing the pull task to define this method with no implementation that it doesn't care about, I made the method have a default no-op implementation that the pull task could ignore and the other tasks could overwrite as before.